features/networking/ovn_winc.feature

```diff
@@ -0,0 +1,56 @@
+Feature: OVNKubernetes Windows Container related networking scenarios
+
+  # @author anusaxen@redhat.com
+  # @case_id OCP-26360
+  @admin
+  Scenario: Ensure Pods and Service communication across window and linux nodes
+    Given the env has hybridOverlayConfig enabled
+    And the env is using windows container
+    Given I have a project
+    And I obtain test data file "networking/list_for_pods.json"
+    When I run the :create client command with:
+      | f | list_for_pods.json |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+      | name=test-pods |
+    And evaluation of `pod` is stored in the :linux_pod clipboard
+    
+    Given I use the "test-service" service
+    And evaluation of `service.ip(user: user)` is stored in the :linux_service_ip clipboard
+    # Run level needs to set to 1 for windows pod to be spawned in a test project
+    When I run the :label admin command with:
+      | resource | namespace                |
+      | name     | <%= project.name %>      |
+      | key_val  | openshift.io/run-level=1 |
+    Then the step should succeed
+    Given I obtain test data file "networking/windows_pod_and_service.yaml"
+    When I run the :create client command with:
+      | f | windows_pod_and_service.yaml |
+    Then the step should succeed
+    And a pod becomes ready with labels:
+      | app=win-webserver |
+    And evaluation of `pod` is stored in the :windows_pod clipboard
```

Comment:
We can save `pod.name` to the clipborad, so we can use `cb.windows_pod` in below steps.